### ISSUE TITLE
why can't link some folder like ProjectSetting?

### ISSUE THREAD

**Issue by chiuan** [svermeulen/Projeny#42]
because i wanna a shared AssetBundles folder under the [ProjectRoot].
how to make this ?

**Comment by svermeulen** [comment]
You're right, this makes sense as a feature.  There should be a way to specify `AssetBundlesPath` in the ProjenyProject.yaml file in the same way you can optionally specify `ProjectSettingsPath`
